packages/rulesets/src/spectral/functions/operation-id-noun-conflicting-model-names.ts

```diff
@@ -0,0 +1,22 @@
+// Check if the noun part of an operationId (Noun_Verb) conflicts with any model names provided in the spec.
+
+export const operationIdNounConflictingModelNames = (operationId: any, _opts: any, ctx: any) => {
+  if (operationId === "" || typeof operationId !== "string") {
+    return [];
+  }
+  if (!operationId.includes("_")) {
+    return [];
+  }
+  const path = ctx.path || [];
+  const errors: any = [];
+  const nounPartOfOperationId = operationId.split("_")[0];
+  const swagger = ctx.document.parserResult.data;
```

Comment:
I used to use this in the uitls.ts, but the old usage in the code has been fixed, please use 'ctx?.documentInventory?.resolved' instead,